di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/service/AuthCodeService.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.di.ipv.stub.cred.service;
+
+import com.nimbusds.oauth2.sdk.AuthorizationCode;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class AuthCodeService {
+    private final Map<String, String> authCodes;
+
+    public AuthCodeService() {
+        this.authCodes = new ConcurrentHashMap<>();
+    }
+
+    public void persist(AuthorizationCode authCode, String resourcePayload) {
```

Comment:
I think this would be better if the authCode was a sting here, making the `put` and `get` symmetrical.

Reply:
But not a blocker.